queries.js

```diff
@@ -0,0 +1,43 @@
+const pgp = require('pg-promise')()
+const connectionString = 'postgres://localhost:5432/sweethornbillbookstore'
+const db = pgp(connectionString)
+
+
+
+const getAllBooks = () => db.any( 'SELECT * FROM books' )
+
+const getSingleBook = ( request, response ) => {
+  const id = request.params.id
+  return db.one('SELECT * FROM books WHERE id = $1', id)
+}
+
+const getSingleAuthor = ( request, response ) => {
+  const author = request.params.author.toLowerCase()
+    console.log(author)
+    return db.one('SELECT * FROM books WHERE LOWER(author) = $1', author)
+}
+
+const getSingleGenre = ( request, response ) => {
+  const genre = request.params.genre
+    return db.one('SELECT * FROM books WHERE genre = $1', genre)
+}
+
+const addBook = ( request, response ) => {
+  return db.any('SELECT * FROM books')
```

Comment:
i changed the variable so that it reflected it's functionality. I guess i could get rid of this and just use my getAllBooks function